_includes/configuration/available-context-menu-items.md

```diff
@@ -0,0 +1,22 @@
+## Available context menu items
+
+The following table shows all context menu items, including items provided by plugins.
+Any menu item can also be added to the context menu.
+To retrieve a list from the editor, run the following command from the browser console:
+
+```js
+tinymce.activeEditor.ui.registry.getAll().contextMenus
+```
+
+| identifier  | Core/Plugin  | Description  |
+| ----------- | ------------ | ------------ |
+| image | [Image]({{site.baseurl}}/plugins/image/) | Adds the _Image..._ item for opening the **Insert/Edit Image** dialog. |
+| imagetools | [Image Tools]({{site.baseurl}}/plugins/imagetools/) | Adds the _Edit image_ item for opening the **Edit Image** dialog. |
```

Comment:
Interesting - this should be "Edit image..." so if not we have a UI string bug

Reply:
![image](https://user-images.githubusercontent.com/14833627/82285078-e65e8e00-99dd-11ea-89dc-73a5518e2b1b.png)
ok. I found a bug!